src/neo/SmartContract/Native/NativeContract.cs

```diff
@@ -28,7 +28,7 @@ public abstract class NativeContract
         public static OracleContract Oracle { get; } = new OracleContract();
         public static DesignateContract Designate { get; } = new DesignateContract();
 
-        [ContractMethod(0, CallFlags.None)]
+        [ContractMethod(180, CallFlags.None)]
```

Comment:
I think it won't be a problem. If you want to call `name()`, you have to push the scripthash onto the stack.

Reply:
I think that we should charge it like a regular push, you should pay the same as `PUSH name(); PUSH neo` otherwise you pay one push and make two.